packages/preview/src/browser/preview-preferences.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { interfaces } from "inversify";
+import {
+  createPreferenceProxy,
+  PreferenceProxy,
+  PreferenceService,
+  PreferenceContribution,
+  PreferenceSchema
+} from "@theia/core/lib/browser";
+
+export const PreviewConfigSchema: PreferenceSchema = {
+  type: "object",
+  properties: {
+    "preview.preferToEditor": {
+      type: "boolean",
+      description: "Open the preview instead of the editor by default.",
```

Comment:
the description and the property name are inverse

Reply:
I think it should be `preview.openByDefault` with a default value `false`.

Reply:
'fine with renaming.

about the default value, I though we want to open the preview preferably to the editor in general. but I'm also fine with making it this way. just asking, I got it the other way around.

Reply:
Ah yes. Let's try opening the preview by default.

Reply:
renaming done.